.rubocop.yml

```diff
@@ -3,9 +3,6 @@ inherit_from: .rubocop_todo.yml
 AllCops:
   TargetRubyVersion: 2.5
```

Comment:
Please update this to 2.7 and also enable new cops + additional plugins. It should look like:

```yaml
require:
  - rubocop-minitest
  - rubocop-performance
  - rubocop-rake

AllCops:
  NewCops: enable
  TargetRubyVersion: '2.7'
```

Note the additional plugins need to be included as a dependency in `Gemfile`.

Reply:
@ekohl okay one more confusion i've, so when I added `rubocop-minitest` and rest of the `gems`.  I can see around 150 offenses created and mostly get generated from `minitest` only. I want to know, How I'm suppose to fix it? Using `auto-correct` isn't always right is it or there is some specific cases to use it? Because if I go and check manually it'll take a lot more time.